Library/Formula/par.rb

```diff
@@ -1,11 +1,9 @@
 # encoding: UTF-8
-require 'formula'
-
 class Par < Formula
-  homepage 'http://www.nicemice.net/par/'
-  url 'http://www.nicemice.net/par/Par152.tar.gz'
-  version '1.52'
-  sha1 '4b83d2ec593bb45ee46d4b7c2bfc590e1f4a41a8'
+  homepage "http://www.nicemice.net/par/"
+  url "http://www.nicemice.net/par/Par152.tar.gz"
+  version "1.52"
+  sha1 "4b83d2ec593bb45ee46d4b7c2bfc590e1f4a41a8"
 
   # A patch by Jérôme Pouiller that adds support for multibyte
   # charsets (like UTF-8), plus Debian packaging.
```

Comment:
Upstream is aware of this patch, it’s linked on their homepage. I don’t know why they haven’t applied it yet.